aiopurpleair/helpers/validator/__init__.py

```diff
@@ -11,4 +11,4 @@ def validate_timestamp(value: int) -> datetime:
     Returns:
         A parsed datetime.datetime object (UTC).
     """
-    return datetime.utcfromtimestamp(value)
+    return datetime.fromtimestamp(value, tz=timezone.utc).replace(tzinfo=None)
```

Comment:
Why are we replacing UTC at the end?

Reply:
Without it, it would be an API change to a timezone aware datetime.
E.g. if someone is using that value to calculate a timedelta, that would fail without `replace(tzinfo=None)`.
```py
>>> from datetime import datetime, timezone
>>> datetime.now(tz=timezone.utc) - datetime.utcnow()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can't subtract offset-naive and offset-aware datetimes
```